async-download-rest/src/main/java/org/uniprot/api/async/download/messaging/consumer/MessageConsumer.java

```diff
@@ -101,7 +101,7 @@ public void onMessage(Message message) {
     }
 
     protected boolean isConsumedBefore(R downloadJob) {
-        return fileHandler.areAllFilesPresent(downloadJob.getId())
+        return fileHandler.areIdAndResultFilesPresent(downloadJob.getId())
                 && ERROR != downloadJob.getStatus();
```

Comment:
same as totalEntries, resetting fromIdCount is not essential. Once it starts to retry it will get updated soon.
And keeping the old count will not make any issue (unlike processedCount and updateCount)